### DISCUSSION TITLE
Will NVIDIA Triton Management Service be open-sourced?

### DISCUSSION THREAD

**Discussion by yeahdongcn** [triton-inference-server/server#6248] [Q&A] [answered]
I've been using KServe + Triton for some time and recently saw this [link](https://www.nvidia.com/en-us/ai-data-science/products/triton-management-service/). I'd like to know whether NVIDIA Triton Management Service will be open-sourced and how to migrate from KServe to it. Thanks.

**Comment by the-david-oy** [comment, marked as answer]
TMS has no plans to be open-sourced at this time. To get access through it, you would need to sign up for [NVIDIA AI Enterprise support.](https://www.nvidia.com/en-us/data-center/products/ai-enterprise/)
